src/main/java/iudx/resource/server/metering/util/Constants.java

```diff
@@ -101,4 +101,10 @@ public class Constants {
   public static final String USERID_SUMMARY_WITHOUT_TIME = " userid = '$9' ";
   public static final String PROVIDERID_SUMMARY = " and providerid = '$8' ";
   public static final String PROVIDERID_SUMMARY_WITHOUT_TIME = " providerid = '$8' ";
+
+  public static String DATA_CONSUMATION_DETAIL_QUERY =
+      "SELECT COALESCE(api_count, 0) AS api_count,COALESCE(consumed_data, 0) AS consumed_data FROM (SELECT COUNT(*) "
+          + "AS api_count,SUM(size) AS consumed_data FROM auditing_rs "
+          + "WHERE userid = '$1' AND time BETWEEN '$2' AND '$3' "
+          + "AND api NOT IN ('/ngsi-ld/v1/ingestion', '/iudx/v1/delete','/iudx/v1/upload'))";
```

Comment:
make a constant value for api names it will help full when fs,di will merge in one we just need to change constant value

Reply:
done.